### ISSUE TITLE
bug(MatTooltip): A11y - tooltip role is missing

### ISSUE THREAD

**Issue by nirkai** [angular/components#20593]
#### Reproduction

MatTooltip not added role="tooltip" to element in DOM.

This issue discussed as part of [this](https://github.com/angular/components/pull/4688) issue.
 
 
#### Expected Behavior

By the  [Criterion 1.4.13: Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) the role should be added to the tooltip element.
What behavior were you expecting to see?

#### Environment

Material: 10.2.0

**Comment by crisbeto** [comment]
The tooltip text is announced through the `aria-describedby` attribute, rather than the role.

**Comment by nirkai** [comment]
It's announce indeed..
But the role still have to be there.

You can see it [here ](https://www.digitala11y.com/tooltip-role/#:~:text=Description,tooltip%20after%20a%20short%20delay.).
Couldn't find it right now in the WCAG explicitly.

**Comment by jelbourn** [comment]
@crisbeto we should probably investigate whether adding `role="tooltip"` to the description element and see if that makes any difference for the readers we support. Probably just a new option/api in `AriaDescriber`.

**Comment by crisbeto** [comment]
If adding the role does make a difference, we'll definitely have to make some changes to `AriaDescriber`, because it currently tries to reuse elements that have the same message. That could be invalid if the element has a `role`.
